README.md

```diff
@@ -18,6 +28,43 @@ or with yarn:
 yarn add claude-code-js
 ```
 
+## Authentication
+
+claude-code-js supports two authentication methods:
+
+### 1. API Key Authentication
+The simplest method - just provide your Anthropic API key:
+
+```javascript
+const claude = new ClaudeCode({
+  apiKey: process.env.ANTHROPIC_API_KEY
+});
+```
+
+### 2. OAuth Token Authentication
```

Comment:
There is already an existing section about `OAuth Token Management`. can you please make sure these 2 sections are merged? 
Also the note about the location of credentials in Mac is not correct. It's stored in keychain that other section outlines it